library.go

```diff
@@ -1793,8 +1793,7 @@ func string_split(fnname string, recv_ Value, args Tuple, kwargs []Tuple) (Value
 		} else if fnname == "split" {
 			res = splitspace(recv, maxsplit+1)
```

Comment:
While you're here, for symmetry, could you remove the +1 on this line and in the expression len(res)+1 == max inside splitspace too?